manifest/dev.pointeroffset/RemoveThatKinkFromSteamLinkCS/info.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "RemoveThatKinkFromSteamLinkCS",
+	"id": "dev.pointeroffset.RemoveThatKinkFromSteamLinkCS",
+	"description": "Forces the SteamLink integration in Resonite to be enabled. (C# Rewrite)",
+	"tags": [
+		"Edrakon",
+		"Quest Pro",
+		"Eye Tracking",
+		"Face Tracking",
+		"WiVRn"
+	],
+	"platforms": ["linux","windows"],
+	"additionalAuthors": ["BlueCyro"],
+	"category": "Hardware Integration",
+	"sourceLocation": "https://github.com/PointerOffset/RemoveThatKinkFromSteamLinkCS",
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/PointerOffset/RemoveThatKinkFromSteamLinkCS/releases/tag/v1.0.0",
+			"artifacts": [{
+				"url": "https://github.com/PointerOffset/RemoveThatKinkFromSteamLinkCS/releases/download/v1.0.0/RemoveThatKinkFromSteamLinkCS.dll",
```

Comment:
ResoniteMod.Link appears to be a 404, it also should be pointing to the mod repo instead of a normal website.